test/load.test.ts

```diff
@@ -591,6 +610,10 @@ describe('#mergeDependency()', () => {
             date: '20200413230227'
           }
         };
+      } else if (uri === 'https://packages.fhir.org/hl7.fhir.r4.core') {
+        return { data: TERM_PKG_RESPONSE };
```

Comment:
The `TERM_PKG_RESPONSE` contents don't match the uri `https://packages.fhir.org/hl7.fhir.r4.core` (i.e., we asked for R4 Core and got back FHIR Terminology). Although the test still works, I'd prefer that it be more consistent with how it works in the real world. Could you change the URI to be the terminology one (`https://packages.fhir.org/hl7.terminology.r4`) or create an `R4_CORE_PKG_RESPONSE` that reflects what R4 core would really look like?